docs/howtos/DataPlane Generation - DPG/08methodInputs.mdx

```diff
@@ -133,6 +133,11 @@ public void post(User user);
 
 ## Spread
 
+Please use the *spread* feature with caution.
+- The anonymous model to be spread into operation should have less than 6 settable properties. See [simple methods](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-parameters).
```

Comment:
.NET and Java have guideline on "simple methods" that should not have more than 6 arguments.

Python and TypeScript/JavaScript does not have such guideline (AFAIK). Python has keyword argument. TypeScript does not put optional arguments to method.